src/java.base/linux/classes/sun/nio/fs/LinuxFileSystem.java

```diff
@@ -67,6 +71,115 @@ public Set<String> supportedFileAttributeViews() {
         return SupportedFileFileAttributeViewsHolder.supportedFileAttributeViews;
     }
 
+    private static UnixException catEx(UnixException x, UnixException y) {
```

Comment:
What is the purpose of add a suppressed exception to a UnixException? A UnixException will get re-mapped to a File SystemException so I assume the suppressed exception will be ignored.

Reply:
It was a mistake; fixed in 2157a1480229fa697c824ae814f11a309627d768.